securedrop/server_os.py

```diff
@@ -1,17 +1,30 @@
+import functools
+import subprocess
 from datetime import date
 
 FOCAL_VERSION = "20.04"
-XENIAL_EOL_DATE = date(2021, 4, 30)
 
-with open("/etc/lsb-release", "r") as f:
-    installed_version = f.readlines()[1].split("=")[1].strip("\n")
+
+@functools.lru_cache()
+def get_xenial_eol_date() -> date:
+    return date(2021, 4, 30)
+
+
+@functools.lru_cache()
+def get_os_release() -> str:
+    return subprocess.run(
+        ["lsb_release", "--release", "--short"],
```

Comment:
An addition to apparmor rules for Apache2 is required for this to pass in staging/prod methinks.

Reply:
good call, probably so, I'll verify and adjust.